src/deadline/job_attachments/upload.py

```diff
@@ -203,26 +204,74 @@ def upload_assets(
 
         return (partial_manifest_key, hash_data(manifest_bytes, hash_alg))
 
+    def _gather_upload_metadata(
+        self,
+        manifest: BaseAssetManifest,
+        source_root: Path,
+        file_system_location_name: Optional[str] = None,
+    ) -> tuple[HashAlgorithm, bytes, str]:
+        """
+        Gathers metadata information of manifest to be used for writing the local manifest
+        """
+        hash_alg = manifest.get_default_hash_alg()
+        manifest_bytes = manifest.encode().encode("utf-8")
+        manifest_name_prefix = hash_data(
+            f"{file_system_location_name or ''}{str(source_root)}".encode(), hash_alg
+        )
+        manifest_name = f"{manifest_name_prefix}_input"
+
+        return (hash_alg, manifest_bytes, manifest_name)
```

Comment:
This function doesn't use any instance variable, can it be static?

Reply:
Method isn't even used! Easier to just delete it altogether.

Remember when the manifest file was a hash + _input? It now has an explicit name + timestamp.

Deleted this method!

Reply:
Sure thing, easy refactor.